src/DagData.hpp

```diff
@@ -127,7 +134,7 @@ struct SharedResourceData
 
 struct Dag
 {
-    static const uint32_t MagicNumber = 0xaBD9224f ^ kTundraHashMagic;
+    static const uint32_t MagicNumber = 0xfac92242 ^ kTundraHashMagic;
```

Comment:
Just curious- is there a regular sequence for updating this magic number?

Reply:
"change it to something random"  is the way we've been going.  to go very pure generate something completely new.  often I just change a few characters, as the amount of change happening in the codebase is small so the risk of collision is negligable.